client/styles/components/_toolbar.scss

```diff
@@ -147,6 +147,11 @@
 	}
 	margin-left: #{5 / $base-font-size}rem;
 	font-size: #{12 / $base-font-size}rem;
+	cursor: pointer;
+}
+
+#autorefresh{
```

Comment:
Adding styles should be done with a class selector, not an ID selector. The reason is because of the complicated rules of CSS specificity—just using class selectors makes it way easier.